### ISSUE TITLE
Mute command should check for the user's role list before muting/unmuting

### ISSUE THREAD

**Issue by Slurpcy** [Cog-Creators/Red-DiscordBot#5161]
### What component of Red (cog, command, API) would you like to see improvements on?

Mutes Cog

### Describe the enhancement you're suggesting.

I think red should check for the user's role list whilst muting / unmuting a user and respond with User#1234 is already muted/unmuted in this server. Because when we run mute / unmute it will keep saying muted / unmuted the user all thou they would have been unmuted or muted.

### Anything else?

That's all

**Comment by MeatyChunks** [comment]
If a member is already muted and you mute them again, it updates the mute duration with the default time or the specified value. You'll see the duration change with `[p]activemutes`. I personally like that it updates to the last used duration if there's a mod conflict, as it can be easily rectified.

**Comment by Slurpcy** [comment]
Yes ur right, in that case it could say user muted indefinitely, but for the unmute cmd I believe it should be changed to member is already unmuted

**Comment by Jackenmen** [comment]
I think this would make sense for `[p]unmute` and it was indeed a cause of confusion:
https://discord.com/channels/133049272517001216/133081046869737472/858494031394635826
